### PR TITLE
AWS v18.3.0

### PR DESCRIPTION
**Pull request by njuettner**
Issue: https://github.com/giantswarm/roadmap/issues/2247

### Checklist
- [x] Roadmap issue created
- [x] Release uses latest stable Flatcar
- [x] Release uses latest Kubernetes patch version

### CODE REVIEW

**Comment by njuettner** [comment]
@giantswarm/team-phoenix the readme will be updated with some information, I just want to let you know tests are passing. Please let me know if I missed anything (app needs to updated)

**Comment by njuettner** [comment]
@giantswarm/team-phoenix PTAL 🙏🏻

**Comment by whites11** [comment]
Unfortunately 1 cncf test is failing and I dunno why because tekton failed to download the results.
I love our e2e infra, did I tell you already?
🙄

**Comment by whites11** [comment]
false alarm, cncf tests green on second attempt

**Review by whites11 — APPROVED** [review]

File: aws/v18.2.1/release.yaml [hunk 1]
```diff
@@ -74,7 +74,7 @@ spec:
   - name: kubernetes
     version: 1.23.15
   date: "2023-03-02T10:06:00Z"
-  state: active
+  state: deprecated
```
Comment (whites11) [inline aws/v18.2.1/release.yaml:77]: technically we don't do this normally. this is another minor so shouldn't be deprecated

Comment (njuettner) [reply aws/v18.2.1/release.yaml]: Right, I reverted it 👍🏻

**Pull request merged**